static/js/vendor/responsive-tables.js

```diff
@@ -1,4 +1,4 @@
-$(document).ready(function() {
+jQuery(document).ready(function() {
```

Comment:
Oh, I was confused because jQuery wasn't working but I forgot to run `npm install`. Reverting.